Dockerfile-linux-arm64

```diff
@@ -0,0 +1,120 @@
+FROM ubuntu:jammy
+
+ARG zlib_version
+ARG zlib_hash
+
+ARG libevent_version
+ARG libevent_hash
+
+ARG openssl_version
+ARG openssl_hash
+
+ARG tor_version
+ARG tor_hash
+
+ARG jobs
+
+COPY gpg-keys/tor.gpg /tor.gpg
+COPY gpg-keys/libevent.gpg /libevent.gpg
+COPY gpg-keys/openssl.gpg /openssl.gpg
+COPY patch /patch
```

Comment:
This could just be
```dockerfile
COPY gpg-keys/tor.gpg gpg-keys/libevent.gpg gpg-keys/openssl.gpg patch /
```

Reply:
I had to leave `patch` on its own line because it wouldn't otherwise work :man_shrugging: